docs/koji.md

```diff
@@ -23,6 +23,7 @@ For atomic-reactor container image builds the `image` type is used, and so type-
 Data which is placed here includes:
 
 - `build.extra.image.autorebuild` (boolean): true if this build was triggered automatically; false otherwise
+- `build.extra.image.help` (boolean): filename if this build has a markdown help converted; None otherwise
```

Comment:
Says 'boolean' but should say 'string or null'. Also let's call it 'null' instead of 'None' as this is a JSON object.